### ISSUE TITLE
Digital Miner not mining any modded ores in 1.12.2

### ISSUE THREAD

**Issue by cigayda** [mekanism/Mekanism#7422]
### Issue description

I am having an issue with my Digital Miner in my modded 1.12.2 Java server (runs on Forge; is part of a custom modpack with over 100 mods on it, mostly your standard FTB mods but with a few additional ones). It refuses to mine any modded ores whatsoever despite them having ore dictionary entries (used the dictionary item to confirm they were recognized and had entries). I tried using both the Materials and OreDict filters. And I tried having it mine specific items like Certus Quartz from Applied Energistics 2. When that didn't work, I tried having it mine all ores from AE2. That didn't work either, so I tried having it mine all ores with *ore* as the OreDict filter. Still no modded ores at all, only vanilla ones. Since other people are able to mine modded ores with it, I assume something must be broken on my end. What could be the problem, and how do I fix it?

### Steps to reproduce

1. Place digital miner and set it up with power.
2. Add a Material or OreDict filter to have it mine any modded ore you want.
3. Try to start the miner, and see that it doesn't mine anything modded.

### Minecraft version

1.15.x or earlier (No longer being developed)

### Forge version

 14.23.5.2860

### Mekanism version

Older

**Comment by Diamantino-Op** [comment]
Is the miner in a claim?

**Comment by cigayda** [comment]
I don't know what that means, so probably not. This is my server and I didn't set anything like that.

At any rate, after talking to the Mekanism people on Discord and then speaking with the server host support staff, I tried a bunch of things and then figured out it was oregen related, having to do with some mods being deleted and re-added to resolve some crash issues when the server was first created. The ores were regenerated, but the digital miner doesn't recognize them as existing in these chunks I had tested it in. When I traveled to a completely new and unvisited chunk a ridiculous distance away from spawn where it had been placed, and then placed it down again in the new spot and set it up, it actually started mining modded ores.

This issue has now been resolved. Thank you :)
